server/routers/jobs_router.py

```
@@ -468,112 +575,29 @@ def load_fiche_metier():
-                    emploiReglemente=fiche.get("emploiReglemente"),
-                    nb_offre=fiche.get("nb_offre"),
-                    liste_salaire_offre=fiche.get("liste_salaire_offre")
-                )
+                    code = fiche.get("code"),
+                    libelle = fiche.get("libelle"),
+                                )
                 try:
                     session.add(fiche_insert)
                 except Exception as e:
-                    logger.exception(f"Erreur lors de l'ajout de la fiche {fiche.get('code')}: {e}")
+                    logger.exception(f"Erreur lors de l'ajout du métier {fiche.get('code')}: {e}")
```

Ruff (0.14.13)

Redundant exception object included in `logging.exception` call

(TRY401)